templates/ts-apollo-mongodb-datasync-backend/README.md

```diff
@@ -0,0 +1,47 @@
+## Graphback Apollo+MongoDB Data Synchronization Runtime Template
+
+Template showcases Graphback Data Synchronization capabilities using
+resolver layer created in-memory as opposed to codegeneration.
+
+Serverless example can be build as docker image with configurable volume for models. 
+This models will be processed at application at startup giving fully featured GraphQL Server 
+following Graphback CRUD specification.
+
+For more information please refer to: 
+
+### Running example
+
+The project has been created using `graphback`. Run the project using the following steps.
+
+- Start the database
+
+```
+docker-compose up -d
+```
+
+- Define your schema in the `model/datamodel.graphql` file. Or use the default:
+
+```
```

Comment:
should we add the annotations here

I am thinking of `@model`, `@datasync` and friends?

Reply:
Added now,  but the example is a bit janky now, I couldn't decide between putting in both models versus putting a different model here, so I only put one model, what do you think?

Reply:
Not bad to call it out in the readme. You can copy [Comment model]( https://github.com/aerogear/graphback/pull/1570/files#diff-e4d6686600db9bde9b351b6328b09c7eR19) here and it will be fine by me.